### DISCUSSION TITLE
How to start plain from lsp-mode specific commit

### DISCUSSION THREAD

**Discussion by Zecat** [emacs-lsp/lsp-mode#3383] [Q&A]
I'm trying to debug something yet unreleased, I would need a `lsp-start-plain.el` equivalent that relies on a specific commit of lsp-mode instead of the melpa version, could someone help me achieve that ?

**Comment by yyoncho** [comment]
One way to do this is to use straight and pick the version you want to use. https://github.com/raxod502/straight.el 
Alternatively, there is `package-install-from-buffer` which can be called from the dir folder you want to install lsp-mode from.

**Reply by Zecat** [reply]
Thank you, I'll figure out using straight to make a reproducible code snippet and close this as soon as I get something working.
